templates/netlify/remix.init/index.js

```diff
@@ -0,0 +1,49 @@
+const inquirer = require("inquirer");
+const fs = require("fs/promises");
+const { join } = require("path");
+
+const files = [
+  ["README.md"],
+  ["netlify.toml"],
+  ["edge-server.js", "server.js"],
+  ["remix.config-edge.js", "remix.config.js"],
+  ["vscode.json", join(".vscode", "settings.json")],
+  ["app/entry.server.tsx"],
+  ["app/root.tsx"],
```

Comment:
Thinking about these ones, can't we just replace `@remix-run/node` with `@remix-run/deno` in these files instead?
Same for `remix.env.d.ts` (which is _not_ included in this list for some reason).

Having a copy of the whole file _could_ cause inconsistencies between `netlify` & `netlify-edge` & causes extra maintainance.

Reply:
@MichaelDeBoey, should I do a regex to replace the import after reading the file and save the updated file via `fs.writeFileSync`? That's how @mcansh did it in our Edge template repo for updating a file.

https://github.com/netlify/remix-edge-template/commit/cdc1ac515d887ad128d7df6e6199223a2f1af717#diff-47d697bbfe7441205f5569284242c952cab65c0d9564344a81c9f10d42673ec3

Or is there something more elegant baked into Remix for stuff like this, e.g. codemod?

Reply:
I think you can do something similar like I do in https://github.com/remix-run/indie-stack/pull/63

This basically boils down to: reading the file content, do a `replace` on `@remix-run/node`, write the file content to the same path

https://github.com/remix-run/indie-stack/blob/7c2506f60123e5242e6d1a47b9cdd7cfd638155d/remix.init/index.js#L10-L23
https://github.com/remix-run/indie-stack/blob/7c2506f60123e5242e6d1a47b9cdd7cfd638155d/remix.init/index.js#L25-L32